src/components/shared/utils/routes/routes.ts

```diff
@@ -1,20 +1,74 @@
-import { URLConstants } from '@deriv-com/utils';
+const getDerivComDomain = () => {
```

Comment:
@mayuran-deriv since all the functions look the same. 
Can we combine all them and make one reusable function ?